src/config.ts

```diff
@@ -1,19 +1,27 @@
-import { withOptions } from '@storybook/addon-options';
-import { addDecorator, configure } from '@storybook/react';
+import { addParameters, configure } from '@storybook/react';
+import { create } from '@storybook/theming';
 
-addDecorator(
-  withOptions({
-    name: pkg.name || 'Stoplight UI-Kit',
-    url: pkg.url || 'https://github.com/stoplightio/ui-kit',
-    goFullScreen: false,
-    showStoriesPanel: true,
-    showAddonPanel: true,
+// This is kinda fragile as it makes the assumptions that it is installed in
```

Comment:
Yeah, it's fragile :P

Reply:
I couldn't think of something better though.